cockatrice/src/deck_loader.cpp

```diff
@@ -200,9 +200,19 @@ void DeckLoader::saveToStream_DeckZoneCards(QTextStream &out, const InnerDecklis
 QString DeckLoader::getCardZoneFromName(QString cardName, QString currentZoneName)
 {
     CardInfo *card = db->getCard(cardName);
-    if(card && card->getIsToken())
+    if (card && card->getIsToken())
```

Comment:
I really need to get clang-format set up. It's not hard to run, the hard part is just tuning its configuration flags to be as close as possible to current style, so as not to rewrite the whole codebase